packages/cli-e2e/tsconfig.json

```diff
@@ -7,7 +7,6 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "forceConsistentCasingInFileNames": true,
-    "types": ["jest"],
-    "lib": ["dom"]
```

Comment:
I added it to this PR, but it is finally unnecessary.
It also causes the build to fail.